src/main/js/App.js

```diff
@@ -151,6 +165,23 @@ const App = () => {
         .catch(console.log);
     };
 
+    const retrieveSurveyNames = () => {
+            fetch(`${webUrl}retrieveSurveyNames`)
+            .then(res => {
+                if (res.status === 200) {
+                    return res.json();
+                } else {
+                    throw res;
+                }
+            })
+            .then(function (data) {
```

Comment:
Please use arrow function syntax instead of function syntax

Reply:
I have made the changes.

As a side note, I think there are advantages to using the function syntax shown here from a readability and style perspective. For starters, it makes it very clear to a reader that they are looking at a function, even for someone who is not familiar with javascript. This makes it slightly easier to visually parse what is going on. The condensed syntax leads to risks of having trouble tracking the meaning in the clutter. Furthermore, the arrow syntax was only introduced in ES6, so the other syntax is more backwards compatible.

The downside is that the function syntax introduces inconsistencies due to the behaviour of `this`, so you have to worry slightly less about calling `bind`. I think that most libraries like `jquery` are designed so that it is possible to get away with not relying on `bind` to deal with callbacks, so the greater consistency may force better programming. Though of course, this does not rely on the style of arrow functions, but rather the specification of the function.

Nevertheless, to improve code consistency, I have update the calls to arrow syntax.